### PR TITLE
Make autoload_js load CSS before JavaScript

### LINKED ISSUE #8681: autoload_js should load CSS before JavaScript
Otherwise layout (among other things) won't have stylesheets available when needed, resulting in wrong layouts. This is a timing-related issue, so depending on e.g. code arrangement in notebook cells or delay between evaluating cells, one may get either good or bad behavior.

### PR DESCRIPTION
**Pull request by mattpap**
I tested this manually in Jupyter notebook and lab.

fixes #8681

### CODE REVIEW

File: bokeh/core/_templates/autoload_js.js [hunk 1]
```diff
@@ -83,23 +105,26 @@ calls it with the rendered model.
...
-    {% endfor -%}
     function(Bokeh) {
-      {%- for url in css_urls %}
-      console.log("Bokeh: injecting CSS: {{ url }}");
-      Bokeh.embed.inject_css({{ url|json }});
-      {%- endfor %}
       {%- for css in css_raw %}
-      console.log("Bokeh: injecting raw CSS");
-      Bokeh.embed.inject_raw_css({{ css }});
+        console.log("Bokeh: injecting raw CSS");
```
Comment (bryevdv) [inline bokeh/core/_templates/autoload_js.js]: As long as we are editing this file can we remove the log statements, or lower them to `debug`

**Comment by bryevdv** [comment]
I've kicked the appveyor build

**Comment by mattpap** [comment]
> I've kicked the appveyor build

How do you do that? According to the documentation there should be a restart button, but I can't find any.

**Comment by bryevdv** [comment]
Currently you need to log in with the `bokeh-integrations` service account. I use a different browser to make isolating that easier. But let me see if I can just invite your personal account as an admin.

**Comment by bryevdv** [comment]
You should have an email invite (I didn't see an option to invite by GH username)

**Comment by mattpap** [comment]
That works. Thanks.

**Pull request merged**